Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_info_fragment.xml

```diff
@@ -0,0 +1,207 @@
+                android:layout_marginTop="@dimen/spacing_medium"
+                android:importantForAccessibility="no"
+                android:src="@drawable/ic_qr_code_illustration"
+                app:layout_constraintStart_toStartOf="@id/trace_location_tracing_icon"
+                app:layout_constraintTop_toBottomOf="@id/trace_location_qr_info_tracing" />
+
+            <TextView
+                android:id="@+id/trace_location_qr_code_illustration_text"
+                style="@style/subtitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_marginEnd="@dimen/spacing_normal"
+                android:text="@string/trace_location_qr_info_qr_code_text"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/trace_location_qr_code_icon"
+                app:layout_constraintTop_toTopOf="@id/trace_location_qr_code_icon" />
+
+            <ImageView
+                android:id="@+id/trace_location_qr_time_sheet_icon"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_medium"
+                android:importantForAccessibility="no"
+                android:src="@drawable/ic_qr_time"
+                app:layout_constraintTop_toBottomOf="@id/trace_location_qr_code_illustration_text"
+                app:layout_constraintStart_toStartOf="@id/trace_location_qr_code_icon" />
+
+            <TextView
+                android:id="@+id/trace_location_qr_time_sheet_text"
+                style="@style/subtitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_marginEnd="@dimen/spacing_normal"
+                android:text="@string/trace_location_qr_info_time_text"
+                app:layout_constraintTop_toTopOf="@id/trace_location_qr_time_sheet_icon"
+                app:layout_constraintStart_toEndOf="@id/trace_location_qr_time_sheet_icon"
+                app:layout_constraintEnd_toEndOf="parent"/>
+
+            <include
+                android:id="@+id/privacy_card"
+                layout="@layout/trace_location_privacy_card"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_medium"
+                android:layout_marginStart="@dimen/spacing_small"
+                android:layout_marginEnd="@dimen/spacing_small"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/trace_location_qr_time_sheet_text" />
+
+            <View
```

Comment:
Please use our custom view `MoreInformationView` for the Data Privacy Button :)

Reply:
Ok, wasn't aware of it.